resources/views/livewire/comment.blade.php

```diff
@@ -20,6 +20,34 @@ class="comment__datetime"
                     </li>
                 @endif
 
+                <li class="comment__toolbar-item">
+                    <button
+                        class="post__quote"
+                        title="{{ __('forum.quote') }}"
+                        x-on:click="
+                            input = document.getElementById(
+                                '{{ $comment->isParent() ? 'new-comment__textarea' : 'reply-comment' }}'
+                            );
+                            input.value +=
+                                '[quote={{ \htmlspecialchars('@' . ($comment->anon ? 'Anonymous' : $comment->user->username)) }}]';
```

Comment:
"Anonymous" is a valid username, and probably shouldn't be pinged here. Maybe drop the `@`?

Reply:
Done: https://github.com/HDInnovations/UNIT3D-Community-Edition/pull/4424/commits/e4001cb1d20f16ca3c558e7fb7bfe91a6d62ab3a